frontend/src/pages/Register.tsx

```diff
@@ -25,12 +31,17 @@ export const Register = () => {
   
         if (!response.ok) {throw new Error("Error registering user");}
         
-        //Redirect to login page after successful registeration
-        alert("Rgistration successful! Please log in.");
-        navigate("/");
+        //direct to home page after successful registeration
+        alert("Rgistration successful!");
+        navigate("/home");
         
       } catch (error) {
-        alert("Error registering")
+        alert("Error registering");
```

Comment:
```suggestion
        console.error("Error registering user:", error);
        alert("Error registering");
```

Reply:
It is a good practice to console the error, so in case it fails, developers can understand easily what is wrong getting feedback on the console.